tic-tac-toe-app/Board.jsx

```diff
@@ -0,0 +1,39 @@
+const Board = (props) => {
+  const [state, setState] = React.useState([]);
```

Comment:
When using `useState()`, the variables used for `state` and `setState` can be named to fit the apps particular context. This ☝️ is certainly valid. Consider refactoring, though, so that the variable names are clear describe the piece of state currently being set.